src/routes.js

```diff
@@ -1417,6 +1420,101 @@ module.exports = async (
     sentPayment.write(paymentRequest);
   });
 
+  app.post("/api/lnd/trackpayment", async (req, res) => {
+    const { router } = LightningServices.services;
+    const { paymentHash, inflightUpdates = true } = req.body;
+    if (req.limituser) {
+      const health = await checkHealth();
+      if (health.LNDStatus.success) {
+        res.sendStatus(403);
+      } else {
+        res.status(500);
+        res.json({ errorMessage: "LND is down" });
+      }
+    }
+
+    logger.info("Tracking payment payment", { paymentHash, inflightUpdates });
+    const trackedPayment = router.trackPaymentV2({
+      payment_hash: paymentHash,
+      no_inflight_updates: !inflightUpdates
+    });
+
+    trackedPayment.on("data", response => {
+      if (response.payment_error) {
+        logger.error("TrackPayment Info:", response)
+        return res.status(500).json({
+          errorMessage: response.payment_error
+        });
+      }
+      
+      logger.info("TrackPayment Data:", response);
+      return res.json(response);
+    });
+
+    trackedPayment.on("status", status => {
+      logger.info("TrackPayment Status:", status);
+    });
+
+    trackedPayment.on("error", async err => {
+      logger.error("TrackPayment Error:", err);
+      const health = await checkHealth();
+      if (health.LNDStatus.success) {
+        res.status(500).json({
+          errorMessage: sanitizeLNDError(err.message)
+        });
+      } else {
+        res.status(500);
+        res.json({ errorMessage: "LND is down" });
+      }
+    });
+  });
+
+  app.post("/api/lnd/sendtoroute", async (req, res) => {
+    const { router } = LightningServices.services;
+    const { paymentHash, route } = req.body;
+    if (req.limituser) {
+      const health = await checkHealth();
+      if (health.LNDStatus.success) {
+        res.sendStatus(403);
```

Comment:
Yeah, I'll have to add a return here to stop the function from proceeding over to running the sendtoroute function on lnd in case if there's any lnd status error returned

Reply:
Code is still there though?